src/components/landing_page/LandingPage.scss

```diff
@@ -0,0 +1,41 @@
+.LandingPage {
+  width: 100%;
+  height: calc(100% - var(--footer-height));
+}
+
+.Content {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  font-family: 'Source Sans Pro', sans-serif;
+  font-weight: 200;
+
+  h1 {
+    color: var(--accent-color-blue);
+    font-size: 4em;
+  }
+
+  .subtitle {
+    height: 4em;
+    text-align: center;
+    font-size: 1.9em;
+    line-height: 2em;
+  }
+}
+
+.Particles {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+}
+
+.unselectable {
+  -webkit-touch-callout: none;
+  -webkit-user-select: none;
+  -khtml-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;

```

Comment:
SCSS applies all the prefixes automatically. `user-select: none;` should be enough.